examples/blur-window/README.md

```diff
@@ -0,0 +1,23 @@
+# Web Page Going to the Background
```

Comment:
We definitely need a better title than simply 'page going to the background' (by itself?).

`Simulate browser window's blur`? `Simulate focusing a different window`?

@Farfurix @DIRECTcut

Reply:
User case: `Simulate the case when the page going to the background`.

Reply:
Since we use the `blur` event to call the `onblur` handler, we can name this case as "Simulate browser window's blur". However, we should point out possible scenarios: focusing a different window, browser going to background.

I also think the current ".. Going to the Background" title is the simplest way to describe this case.

Reply:
Maybe we could use some variation of [blur event definition](https://developer.mozilla.org/en-US/docs/Web/API/Window/blur_event)?

> Web Page Losing Focus

Reply:
> "Simulate browser window's blur"
It's the wrong caption. An example should be named as user scenario - `Simulate the case when the page going to the background`.

Raising of a blur event is implementation details.

Reply:
`Simulate the Web Page Losing Focus` is also fine. I feel like 'going to the background' is a bit ambiguous.

Reply:
I think @VasilyStrelyaev's option is the best one. Sometimes the browser window and the new active window are small and can be together on the sreen's foreground (e.g. left half and right half). Also I don't feel that a page can be switched to the background. A window can which embodies the page can.

Reply:
That's @DIRECTcut's but thanks!